### DISCUSSION TITLE
upgrade appversion and chart version in chart.yaml

### DISCUSSION THREAD

**Discussion by Sid129** [renovatebot/renovate#19048] [Request Help] [answered]
Hello,

Thank you for the great tool. we are using renovate to upgrade helm charts. we have helm charts with a single dependency. is there a way to upgrade the appVersion and chart version in chart.yml same as the dependency chart?

**Comment by viceice** [comment, marked as answer]
yes, we do that ourself:
https://github.com/renovatebot/helm-charts/blob/c754175e90defed41b7981847330b23fa00b1ea4/renovate.json#L12-L21
